src/scripts/main.js

```diff
@@ -1,7 +1,104 @@
 'use strict';
 
-// Uncomment the next lines to use your game instance in the browser
-// const Game = require('../modules/Game.class');
-// const game = new Game();
+import Game from '../modules/Game.class';
 
-// Write your code here
+const game = new Game();
+
+const button = document.querySelector('.start');
+
+const gameField = document.querySelector('.game-field');
+const gameScore = document.querySelector('.game-score');
+
+const messageStart = document.querySelector('.message-start');
+const messageLose = document.querySelector('.message-lose');
+const messageWin = document.querySelector('.message-win');
+
+button.addEventListener('click', () => {
+  if (button.classList.contains('start')) {
+    game.start();
+
+    button.classList.remove('start');
+    button.classList.add('restart');
+    button.textContent = 'Restart';
+
+    messageStart.classList.add('hidden');
+    messageLose.classList.add('hidden');
+    messageWin.classList.add('hidden');
+  } else if (button.classList.contains('restart')) {
+    game.restart();
+
+    button.classList.remove('restart');
+    button.classList.add('start');
+    button.textContent = 'Start';
+
+    messageStart.classList.add('hidden');
+    messageLose.classList.add('hidden');
+    messageWin.classList.add('hidden');
+  }
+
+  manageGameField();
+  manageGameScore();
+});
```

Comment:
It's good practice to separate concerns by not mixing UI logic with game logic. Consider creating separate functions for updating UI elements based on the game state, and call those functions from within the event listener.